docs/cluster_spec.md

```diff
@@ -1565,3 +1565,48 @@ spec:
 ```
 
 To configure Pods to assume the given IAM roles, enable the [Pod Identity Webhook](/addons/#pod-identity-webhook). Without this webhook, you need to modify your Pod specs yourself for your Pod to assume the defined roles.
+
+# API Changes
+
+kOps is working on updating the `v1alpha2` API to a newer version. That new API
+is still under development, but the internal form of the API and validation error
+messages use the new field names. The following table tracks the changes, excepting
+the removal of fields no longer in use.
+
+| v1alpha2 Field                                         | New Field                                                      |
+|--------------------------------------------------------|----------------------------------------------------------------|
+| additionalSans                                         | api.AdditionalSANs                                             |
+| api.loadBalancer.subnets.allocationId                  | api.loadBalancer.subnets.allocationID                          |
+| api.loadBalancer.useForInternalApi                     | api.loadBalancer.useForInternalAPI                             |
+| cloudConfig.azure                                      | cloudProvider.azure                                            |
+| cloudConfig.azure.subscriptionId                       | cloudProvider.azure.subscriptionID                             |
+| cloudConfig.azure.tenantId                             | cloudProvider.azure.tenantID                                   |
+| cloudConfig.openstack                                  | cloudProvider.openstack                                        |
+| cloudProvider (string)                                 | cloudProvider (map)                                            |
+| DisableSubnetTags                                      | tagSubnets (value inverted)                                    |
+| etcdClusters[*].etcdMembers[*].kmsKeyId                | etcdClusters[*].etcdMembers[*].kmsKeyID                        |
+| etcdClusters[*].etcdMembers[*].volumeIops              | etcdClusters[*].etcdMembers[*].volumeIOPS                      |
```

Comment:
The markdown preview shows these values as getting partially italicized due to the asterisks. We may need to wrap every value in backticks.